index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./index.css" />
+    <link rel="shortcut icon" href="#" type="image/x-icon" />
+    <title>Customer Video Player</title>
```

Comment:
Check out the meaning difference between "custom" and "customer". And choose the correct title.